tests/nightly/apache_rat_license_check/.rat-excludes

```diff
@@ -0,0 +1,54 @@
+.*xml
+\..*
+.*css
+\\.*
+.*ipynb
+.*html
+.*json
+.*js
+.*txt
+.*md
+3rdparty/*
+R-package/*
+src/operator/mkl/*
+trunk/*
+docker/*
+docker_multiarch/*
+.*\\.m
+.*\\.mk
+.*\\.R
+contrib/*
```

Comment:
the contrib folder is ignored?

Reply:
I dont think contrib can have the apache header. 
Again, I am not sure what the best way to handle these is

Reply:
How does other apache projects handle contrib folder? I thought licenses are required

Reply:
@eric-haibin-lin, see my comment at the end of this PR. 
I do not want to add wrong licenses to any files and hence have left these files as is until I am sure they can have the ASF header.